packages/sdk/client-services/src/packlets/spaces/data-space-manager.ts

```diff
@@ -211,6 +200,46 @@ export class DataSpaceManager {
     return space;
   }
 
+  async isDefaultSpace(space: DataSpace): Promise<boolean> {
+    const rootDoc = await this._getSpaceRootDocument(space);
+    const [_, properties] = findPropertiesObject(rootDoc.docSync()) ?? [];
+    return properties?.data?.[defaultKey] === this._signingContext.identityKey.toHex();
```

Comment:
change to `defaultSpaceKey`? (defaultKey could mean many things).

but, could spaces have some generic attributes (e.g., tags?) -- that other layers of the system can interpret as a "default space" -- rather than tying together these concepts directly? E.g., in Gmail, folders are represented as tags on a message, and there are special "system" tags for Inbox, Sent, Deleted, etc. So Sent is treated as a "special" folder by some subsystems, but the folders are all basically the same (just have different tags). It's decoupled and more flexible. (e.g., later perhaps we want "hidden" spaces, or "transient" (in-memory) spaces, etc.